test/concerns/versions_test.rb

```diff
@@ -0,0 +1,8 @@
+require 'test_helper'
+
+class VersionsTest < ActiveSupport::TestCase
+  test '_document_versions return empty hash for blank history' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.